### PR TITLE
Fix bad link formatting

### PR DESCRIPTION
**Pull request by kujaomega**
**Pull Request Requirements:**
This PR removes link formatting from tiptap fixing bad link formatting:
Screenshot from 2024-07-02 21-14-52
The changes adds links to URLs only.

### CODE REVIEW

File: packages/text/src/kits/default-kit.ts [hunk 1]
```diff
@@ -54,11 +54,7 @@ export const DefaultKit = Extension.create<DefaultKitOptions>({
       Highlight.configure({
         multicolor: false
       }),
-      Typography.configure({}),
-      Link.configure({
```
Comment (aonnikov) [inline packages/text/src/kits/default-kit.ts:58]: You cannot just remove this extension:
1. Link button in toolbar will be broken
2. Existing content that contains links will be broken

File: packages/presentation/src/components/markup/NodeContent.svelte [hunk 2]
```diff
@@ -49,7 +100,9 @@
   {#if node.type === MarkupNodeType.doc}
     <MarkupNodes {nodes} {preview} />
   {:else if node.type === MarkupNodeType.text}
-    {node.text}
+    <p>
```
Comment (aonnikov) [inline packages/presentation/src/components/markup/NodeContent.svelte:103]: adding additional paragraphs may affect layout

File: packages/presentation/src/components/markup/NodeContent.svelte [hunk 3]
```diff
@@ -49,7 +100,9 @@
...
+      {@html urlify(node.text)}
```
Comment (aonnikov) [inline packages/presentation/src/components/markup/NodeContent.svelte:104]: `@html` is potentially dangerous , we are trying not to use it

**Review by aonnikov — CHANGES_REQUESTED** [review]

File: packages/presentation/src/components/markup/NodeContent.svelte (hunk 3 above)
Comment (kujaomega) [reply packages/presentation/src/components/markup/NodeContent.svelte:104]: urlify function, encodes strings to html encoded characters except added links

**Pull request closed without merging**